docs/concepts/vega-chain/validator-scores-and-rewards.md

```diff
@@ -65,7 +65,9 @@ The **raw validator score** is calculated as follows:
 
 `raw_validator_score` = (`validator_stake_i` - `flat_penalty` - `higher_penalty`) / `total_stake`
 
-In other words, the network calculates an optimal stake that represents an even distribution of stake for the current number of consensus validators and the desired competition level. It then penalises any validators that have stake that exceeds that amount. The raw validator score is then the resulting amount divided by the total stake on the network.
+In other words, the network calculates an optimal stake that represents an even distribution of stake for the current number of consensus validators and the desired competition level*. It then penalises any validators that have stake that exceeds that amount. The raw validator score is then the resulting amount divided by the total stake on the network.
```

Comment:
'... that exceeds that amount by a too large margin' (we don't want completely even distribution, but no whales)